Proton/Sources/Swift/Attachment/Attachment.swift

```diff
@@ -126,16 +126,21 @@ open class Attachment: NSTextAttachment, BoundsObserving {
     /// ```
     public var string: NSAttributedString {
         guard let isBlockAttachment = isBlockAttachment else { return NSAttributedString(string: "<UNKNOWN CONTENT TYPE>") }
```

Comment:
Local variable `isBlockAttachment ` is not used
```suggestion
        guard isBlockAttachment != nil else { return NSAttributedString(string: "<UNKNOWN CONTENT TYPE>") }
```